src/nplinker/pairedomics/podp_antismash_downloader.py

```diff
@@ -0,0 +1,423 @@
+import csv
+import os
+import re
+import time
+import zipfile
+import httpx
+from deprecated import deprecated
+from bs4 import BeautifulSoup
+from progress.bar import Bar
+from nplinker.logconfig import LogConfig
+
+
+logger = LogConfig.getLogger(__name__)
+
+# urls to be given to download antismash data
+ANTISMASH_DB_PAGE_URL = 'https://antismash-db.secondarymetabolites.org/output/{}/'
+ANTISMASH_DB_DOWNLOAD_URL = 'https://antismash-db.secondarymetabolites.org/output/{}/{}'
+
+# The antiSMASH DBV2 is for the availability of the old version, better to keep it.
+ANTISMASH_DBV2_PAGE_URL = 'https://antismash-dbv2.secondarymetabolites.org/output/{}/'
+ANTISMASH_DBV2_DOWNLOAD_URL = 'https://antismash-dbv2.secondarymetabolites.org/output/{}/{}'
+
+NCBI_LOOKUP_URL_NEW = 'https://www.ncbi.nlm.nih.gov/assembly/?term={}'
+
+JGI_GENOME_LOOKUP_URL = 'https://img.jgi.doe.gov/cgi-bin/m/main.cgi?section=TaxonDetail&page=taxonDetail&taxon_oid={}'
+
+USER_AGENT = 'Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:86.0) Gecko/20100101 Firefox/86.0'
+
+class GenomeStatus:
+
+    def __init__(self, original_id, resolved_id, attempted=False, filename=""):
+        self.original_id = ';'.join(original_id.split(','))
+        self.resolved_id = None if resolved_id == 'None' else resolved_id
+        self.attempted = True if attempted == 'True' else False
+        self.filename = filename
+
+    @classmethod
+    def from_csv(cls, original_id, resolved_id, attempted, filename):
+        return cls(original_id, resolved_id, attempted, filename)
+
+    def to_csv(self):
+        return ','.join([
+            str(self.original_id),
+            str(self.resolved_id),
+            str(self.attempted), self.filename
+        ])
```

Comment:
is this class used also somewhere else? If so, I think it should move to its own file.

Reply:
It's used only in this module